docs/source/algorithms/HYSPECScatteredTransmissionCorrectionMD-v1.rst

```diff
@@ -0,0 +1,69 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+The intensity :math:`I` of every event collected is corrected by:
```

Comment:
Please replace the first sentence by "In instruments where there is a polarization analysis stage in the scattered beam, neutron intensity is reduced by an absorption factor. This has an exponential dependence on the scattered energy. To correct for this, the detected intensity :math:`I` of every event collected is corrected by:"